File-Formats/PDF/Working-with-OCR/Docker.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Performing OCR in Docker | Syncfusion
+description: Learn how to Perform OCR in Docker with easy steps using Syncfusion .NET OCR library.
+platform: file-formats
+control: PDF
+documentation: UG
+keywords: Assemblies
+---
+
+# How to Perform OCR on a PDF Document Using Docker container
+
+Optical Character Recognition (OCR) is a technology that converts scanned paper documents from PDF files or images into searchable and editable data.
+
+The [Syncfusion OCR processor library](https://www.syncfusion.com/document-processing/pdf-framework/net/pdf-library/ocr-process) has extended support to OCR process PDF documents and other scanned images in [docker](https://www.docker.com/why-docker/) container with the help of the [Tesseract](https://github.com/tesseract-ocr/tesseract) OCR engine.
+
+## Steps to perform OCR on a PDF document programmatically
```

Comment:
Steps to perform OCR on entire PDF document in Docker

Reply:
Changed the content